src/test/java/com/example/bank/transfer/export/XlsxTransferHistoryGeneratorAssert.java

```diff
@@ -1,98 +0,0 @@
-package com.example.bank.transfer.export;
-
-import com.example.bank.transfer.model.TransferHistory;
-import lombok.RequiredArgsConstructor;
-import org.apache.poi.ss.usermodel.*;
-import org.assertj.core.api.Assertions;
-import org.assertj.core.api.SoftAssertions;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.stream.IntStream;
-
-import static com.example.bank.transfer.export.WorkbookCreator.headerCellTitles;
-import static com.example.bank.transfer.export.WorkbookCreator.transferHistorySheetName;
-import static org.junit.jupiter.api.Assertions.*;
-
-@RequiredArgsConstructor
```

Comment:
assertThat() powinna być publiczna

- . Brak konstruktora – Można dodać private
-  hasValidSheetName() – Można dodać lepszy komunikat błędu
- hasValidHeaderName() – Obsługa null dla headerRow
-  hasValidRowDataValues() – Sprawdzenie czy rzędy istnieją
- hasValidHeaderNumber() – Zmiana kolejności asercji -> assertAll(
    () -> assertEquals(lastCellNum, expectedLastCellNum, "Header column count does not match expected."),
    () -> assertEquals(testHeaderIndex, headerIndex, "Header index does not match expected.")
);

-